scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -131,68 +128,214 @@ def make_create_workspace_request(workspace_name, auth_domains, namespace=NAMESP
     return create_ws_request
 
 
+def copy_workspaces_workflows(destination_workspace_namespace, destination_workspace_name, source_workspace_namespace, source_workspace_name):
```

Comment:
the function says that it copies over multiple workspaces' workflows but it is only for one workspace